### DISCUSSION TITLE
How to get the note‘s title displayed on iPad?

### DISCUSSION THREAD

**Discussion by sirlaughalot90** [chrisgrieser/shimmering-focus#181] [Q&A] [answered]
I can’t figure it out.. How can I enable the title bar on iPad to change a note‘s title? Seems to be impossible. 🤔

**Comment by chrisgrieser** [comment, marked as answer]
Development of the theme on mobile is on hold right now, since Obsidian mobile will get a complete theme rehaul soon. Most likely, the issue will be adressed by Obsidian core already by then. If not, please come back 🙂
